doc/source/cluster/kubernetes/user-guides/experimental.md

```diff
@@ -0,0 +1,46 @@
+(kuberay-experimental)=
+
+# Experimental Features
+
+We provide an overview of new and experimental features available
+for deployments of Ray on Kubernetes.
+
+## RayServices
+
+The `RayService` controller enables fault-tolerant deployments of
+{ref}`Ray Serve <rayserve>` applications on Kubernetes.
+
+If your Ray Serve application enters an unhealthy state, the RayService controller will create a new Ray Cluster.
+Once the new cluster is ready, Ray Serve traffic will be re-routed to the new Ray cluster.
+
+See the [KubeRay docs on RayServices][KubeServe] for details.
```

Comment:
as we discussed please update link to point at the serve k8s user guide once this is merged:
https://github.com/ray-project/ray/pull/27747

Reply:
Fixed!